lib/admin/WP_Auth0_Admin_Features.php

```diff
@@ -423,19 +423,24 @@ public function security_validation( $old_options, $input ) {
 					__( 'No database connections found for this application. ', 'wp-auth0' ) .
 					$this->get_dashboard_link( 'connections/database', __( 'See all database connections', 'wp-auth0' ) )
 				);
+				$input['password_policy'] = $old_options['password_policy'];
```

Comment:
Nothing changed at Auth0 so we should not change the setting in WordPress either.